tools/fanatec_led_server.py

```diff
@@ -122,38 +141,51 @@ def run(self):
             if not self.tick():
                 break
 
-            if self.revLightsPercent > 0.95:
+            if self._revLightsPercent > 0.95:
```

Comment:
This should be now ` > 95`, right? (This creates a flashing effect when revs are > 95%)

Reply:
yes, fixed!